src/System.Windows.Forms/tests/IntegrationTests/WinformsControlsTest/ComboBoxesWithScrollBars.cs

```diff
@@ -0,0 +1,188 @@
+    {
+        public ComboBoxesWithScrollBars()
+        {
+            InitializeComponent();
+
+            // Init comboboxes
+            for (int i = 0; i <= 40; i++)
+            {
+                comboBox1.Items.Add(i);
+                comboBox2.Items.Add(i);
+                comboBox3.Items.Add(i);
+            }
+        }
+
+        protected override void OnLoad(EventArgs e)
+        {
+            // Set current NumericUpDowns values to heights
+            changeDDH_UpDown1.Value = comboBox1.DropDownHeight;
+            changeCBHeight_UpDown2.Value = comboBox2.Size.Height;
+            changeDDH_UpDown3.Value = comboBox3.DropDownHeight;
+
+            // Set current NumericUpDowns values to MaxDropDownItems values
+            maxDropDownItemsUpDown1.Value = comboBox1.MaxDropDownItems;
+            maxDropDownItemsUpDown2.Value = comboBox2.MaxDropDownItems;
+            maxDropDownItemsUpDown3.Value = comboBox3.MaxDropDownItems;
+
+            // Set current CheckBoxes values to IntegralHeight values
+            integralHeightCheckBox1.Checked = comboBox1.IntegralHeight;
+            integralHeightCheckBox2.Checked = comboBox2.IntegralHeight;
+            integralHeightCheckBox3.Checked = comboBox3.IntegralHeight;
+
+            // Set current CheckBoxes values to DrawMode values
+            useDifferentHeightsCheckBox1.Checked = comboBox1.DrawMode == DrawMode.OwnerDrawVariable;
+            useDifferentHeightsCheckBox2.Checked = comboBox2.DrawMode == DrawMode.OwnerDrawVariable;
+            useDifferentHeightsCheckBox3.Checked = comboBox3.DrawMode == DrawMode.OwnerDrawVariable;
+
+            // Add event handlers
+            changeDDH_UpDown1.ValueChanged += changeDDH_UpDown1_ValueChanged;
+            changeCBHeight_UpDown2.ValueChanged += changeCBHeight_UpDown2_ValueChanged;
+            changeDDH_UpDown3.ValueChanged += changeDDH_UpDown3_ValueChanged;
+            maxDropDownItemsUpDown1.ValueChanged += maxDropDownItemsUpDown1_ValueChanged;
+            maxDropDownItemsUpDown2.ValueChanged += maxDropDownItemsUpDown2_ValueChanged;
+            maxDropDownItemsUpDown3.ValueChanged += maxDropDownItemsUpDown3_ValueChanged;
+            integralHeightCheckBox1.CheckedChanged += integralHeightCheckBox1_CheckedChanged;
+            integralHeightCheckBox2.CheckedChanged += integralHeightCheckBox2_CheckedChanged;
+            integralHeightCheckBox3.CheckedChanged += integralHeightCheckBox3_CheckedChanged;
+            useDifferentHeightsCheckBox1.CheckedChanged += useDifferentHeightsCheckBox1_CheckedChanged;
+            useDifferentHeightsCheckBox2.CheckedChanged += useDifferentHeightsCheckBox2_CheckedChanged;
+            useDifferentHeightsCheckBox3.CheckedChanged += useDifferentHeightsCheckBox3_CheckedChanged;
+
+            base.OnLoad(e);
+        }
+
+        private void changeDDH_UpDown1_ValueChanged(object sender, EventArgs e)
+        {
+            comboBox1.DropDownHeight = (int)((NumericUpDown)sender).Value;
```

Comment:
```suggestion
            comboBox1.DropDownHeight = changeDDH_UpDown1.Value;
```

In fact, since there's only one instruction here, and the handler is exclusive to `changeDDH_UpDown1`, this all can be inlined, thus drastically reducing the code:
```cs
changeDDH_UpDown1.ValueChanged += (s, e) => comboBox1.DropDownHeight = changeDDH_UpDown1.Value;
```

ditto below